### DISCUSSION TITLE
iOS beta testflight

### DISCUSSION THREAD

**Discussion by brickerking** [rgriebl/brickstore#917] [General]
The ios testflight link isn’t working. I was able to use it a few weeks ago, but it expired and now i can’t use the app anymore. Should there be an updated link or am I just out of luck?

**Comment by brickerking** [comment]
Is the ios public Testflight full (not sure of they fill up) or just not available right now? Would be helpful to know instead of checking every day. Thanks!

**Reply by rgriebl** [reply]
Testflight builds are only valid for 90 days and the automatic refresh did not work, because Apple wanted me to manually approve their updated T&C. I've done that now and pushed a new build. Please let me know if it works for you now.

**Reply by brickerking** [reply]
Thanks, that worked! I’m back in. Much appreciated!
